test/perf/dotnet-tools/BenchmarkDotNet.Extensions/BenchmarkDotNet.Extensions.csproj

```diff
@@ -6,8 +6,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="BenchmarkDotNet" Version="0.12.1.1521" />
-    <PackageReference Include="BenchmarkDotNet.Diagnostics.Windows" Version="0.12.1.1521" />
+    <PackageReference Include="BenchmarkDotNet" Version="0.13.1" />
```

Comment:
In a separate PR please add `test/perf` here: https://github.com/PowerShell/PowerShell/blob/master/tools/UpdateDotnetRuntime.ps1#L81

So we will always use the latest.